src/NLog/Targets/DatabaseParameterInfo.cs

```diff
@@ -134,6 +134,13 @@ public DatabaseParameterInfo(string parameterName, Layout parameterLayout)
         [DefaultValue(null)]
         public CultureInfo Culture { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether empty value should translate into DbNull
+        /// </summary>
+        /// <docgen category='Parameter Options' order='8' />
+        [DefaultValue(false)]
+        public bool AllowDbNull { get; set; }
```

Comment:
What about EmptyToDbNull or Nullable?

Reply:
Well in SQL language it is often called allow NULL. So that is why I like the **Allow**-part. But because in DotNet then it is not an actually null-reference but instead the artificial **DbNull**. This gives **AllowDbNull**

Reply:
Clear, but it does more than allow, it translates. I'm not sure how clear that is from the name (see also the description, it's using the term translate and not allow). But I also like to have feedback if others :)

Reply:
Have updated to source-code comment about having allow null configured for the database-column :)